main.go

```diff
@@ -45,6 +45,7 @@ var (
 
 		AutoRevoke bool          `flag:"auto-revoke" vardefault:"auto-revoke" description:"Automatically revoke older certificates for this FQDN"`
 		CertTTL    time.Duration `flag:"ttl" vardefault:"ttl" description:"Set the TTL for this certificate"`
+		OvpnKey    string        `flag:"ovpn-key" vardefault:"secret/ovpn" description:"Specify a secret name that holds an OpenVPN shared key"`
```

Comment:
This is not how `vardefault` works: `vardefault` just contains a key for the map loaded from file / `defaultConfig` in line 56. Please use `ovpn-key` here and put the default into the mapping in line 56.

Reply:
Also default should be `""` as this flag otherwise would break backwards compatibility.